README.md

```diff
@@ -60,10 +60,39 @@ Thanks goes to these wonderful people ([emoji key](https://allcontributors.org/d
 
 <!-- markdownlint-enable -->
 <!-- prettier-ignore-end -->
+
 <!-- ALL-CONTRIBUTORS-LIST:END -->
 
 This project follows the [all-contributors](https://github.com/all-contributors/all-contributors) specification. Contributions of any kind welcome!
 
 ## References
 
 [Akita-Demo](https://github.com/seangwright/akita-demo)
+
+## Build / Run
+
+### Client
+
+- Run the "Client - Install" VS Code task
+- Run the "Client - Serve" VS Code task
+
+- Optional: Run any of the following tasks
+
+  - Client - Build
+
+### Functions
+
+- Ensure the [Azure Functions](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-azurefunctions)
```

Comment:
I couldn't figure out a better DevEx for Azure Functions in VS Code - but this is only a few steps, so that's not too bad.

Debugging & breakpoints work as expected.